torch/nn/modules/module.py

```diff
@@ -986,6 +988,7 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                                       'whose dimensions in the checkpoint are {}, '
                                       'an exception occurred : {}.'
                                       .format(key, param.size(), input_param.size(), ex.args))
+                    raise ex
```

Comment:
This should never raise!

Reply:
some testing code that I forgot to delete ...